--- configure.ac.old	2015-12-02 22:23:48.726073674 -0600
+++ configure.ac	2015-12-02 22:22:48.788098649 -0600
@@ -75,9 +75,10 @@
 AC_SUBST(PULSE_CFLAGS)
 
 # If using GCC specifiy some additional parameters
+# Modified 2015-12-02 by Philip Creighton <phil@pdc302.net> to include compile flags for c++11
 if test "x$GCC" = "xyes" ; then
    CFLAGS="$CFLAGS -pipe -Wall -W -Wno-unused-parameter"
-   CXXFLAGS="$CXXFLAGS -pipe -Wall -W"
+   CXXFLAGS="$CXXFLAGS -pipe -Wall -W -std=c++11"
 fi
 
 ZP_LYNX_DOC
